Cargo.toml

```diff
@@ -99,6 +102,8 @@ starknet_api = "=0.10.0"
 cairo-lang-starknet-classes = "2.6.0"
 cairo-lang-utils = "2.6.0"
 
+alloy = { version = "0.2.0", features = ["full", "json"] }
```

Comment:
the "full" flag here is probably overkill if we only want to support just sending rpc requests

Reply:
resolved